modules/islandora_binary_object_zip_importer/includes/importer.inc

```diff
@@ -208,6 +209,30 @@ class BinaryObjectZipBatchImportObject extends ZipBatchImportObject {
       $datastream['label'] = $this->getLabel($datastream['dsid'], $datastream);
     }
 
+    // See if the user uploaded DC, if not create it now.
+    if (!isset($to_return['DC'])) {
+      $obtained_dc = $this->getDC();
+      if ($obtained_dc) {
+        $name = file_create_filename('DC record.xml', 'public://');
```

Comment:
This call isn't really necessary... Just throw it in the second parameter of the `file_unmanaged_save_data()` call, and it will get made unique as necessary... That said, the same applies [as above](https://github.com/Islandora-Labs/islandora_binary_object/pull/20/files#r76988524): The return of `file_unmanaged_save_data()` should be used instead of `$name`.